<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayForVip</class>
 <widget class="qMRMLWidget" name="PayForVip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>支付窗口</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="PayForVip2" native="true">
     <widget class="QLineEdit" name="lblCode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>260</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblErrorInfo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>155</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 106, 106);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVip">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>224</y>
        <width>260</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>激活</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>激活码：</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qmrmlwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
